test/core/derivatives/lstm_settings.py

```diff
@@ -0,0 +1,30 @@
+"""Test configurations for `backpack.core.derivatives` LSTM layers.
+
+Required entries:
+    "module_fn" (callable): Contains a model constructed from `torch.nn` layers
+    "input_fn" (callable): Used for specifying input function
+
+Optional entries:
+    "target_fn" (callable): Fetches the groundtruth/target classes
+                            of regression/classification task
+    "loss_function_fn" (callable): Loss function used in the model
+    "device" [list(torch.device)]: List of devices to run the test on.
+    "id_prefix" (str): Prefix to be included in the test name.
+    "seed" (int): seed for the random number for torch.rand
+"""
+
+import torch
+
+LSTM_SETTINGS = []
+
+###############################################################################
+#                                   examples                                  #
+###############################################################################
+
+example = {
+    "module_fn": lambda: torch.nn.LSTM(input_size=5, hidden_size=3, num_layers=1),
+    "input_fn": lambda: torch.rand(size=(10, 8, 5)),
+}
+LSTM_SETTINGS.append(example)
+
+# TODO write rest of test similar to linear_settings
```

Comment:
Is this still a TODO?

Reply:
Depends...
There is only one test, so more tests would add robustness. On the other hand, there will be more tests in the extensions and there will be more tests if other hyperparameters are admitted.
What do you think?

Reply:
Fine for now. Can you remove the TODO comment?